versions/a-/azure-c-shared-utility.json

```
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "ae2cd1418fa9f6d28a8e515d972c1eb982ed7df8",
+      "version-date": "2023-08-07",
+      "port-version": 1
```

```suggestion
      "port-version": 0
```
Need to delete the `port-version` field in the `vcpkg.json`, and use the command `./vcpkg x-add-version`